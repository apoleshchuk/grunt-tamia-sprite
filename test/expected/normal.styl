normal_button-menu-alert = 0px 0px 35px 33px
normal_button-menu-disabled = -38px 0px 35px 33px
normal_button-menu-hover = 0px -36px 35px 33px
normal_button-menu-normal = -38px -36px 35px 33px
normal_width = 73px
normal_height = 69px