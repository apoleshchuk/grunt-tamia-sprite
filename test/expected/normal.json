{"properties":{"width":73,"height":69},"coordinates":{"button-menu-alert":{"x":0,"y":0,"width":35,"height":33},"button-menu-disabled":{"x":-38,"y":0,"width":35,"height":33},"button-menu-hover":{"x":0,"y":-36,"width":35,"height":33},"button-menu-normal":{"x":-38,"y":-36,"width":35,"height":33}}}